### PR TITLE
Code coverage fixes

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Motivation and context

- Removed code coverage from full and schedule workflows
- Updated codecov.yml
- Fixed coverage saving in rqworker.py
- Now one code coverage report is sent to codecov after passing all tests

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by yasakova-anastasia** [comment]
@kirill-sizov, could you please review this PR?

File: tests/python/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pytest==6.2.5
 pytest-cases==3.6.13
 pytest-timeout==2.1.0
-pytest-cov==4.0.0
+pytest-cov==4.1.0
```
Comment (kirill-sizov) [inline tests/python/requirements.txt:4]: This comment isn't directly related to this PR, but I'm not quite sure why we need it here.

As I understand we calculate coverage inside our containers and our cvat server has coverage dependency for it.
After coverage is calculated and written we move generated coverage files from containers to the local filesystem and then send it to codecov.

What is the point of having pytest-cov and running tests with `--cov` argument?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -167,12 +167,21 @@ jobs:
...
+          name: coverage_results
+          path: |
+            coverage.json
+            coverage_cvat_server.json
+            coverage_cvat_worker_annotation.json
+            coverage_cvat_worker_import.json
+            coverage_cvat_worker_export.json
+            coverage_cvat_worker_quality_reports.json
+            coverage_cvat_worker_webhooks.json
+            coverage_cvat_utils.json
```
[the remark this answers is not in the document]
Comment (yasakova-anastasia) [reply .github/workflows/main.yml]: Fixed

File: tests/python/requirements.txt (hunk 1 above)
Comment (yasakova-anastasia) [reply tests/python/requirements.txt:4]: We get code coverage for sdk and cli with this.

File: tests/python/rest_api/test_tasks.py [hunk 3]
```diff
@@ -2279,6 +2279,7 @@ def _init_tasks(cls):
 
             cls.data[key] = (task, dataset_file)
 
+    @pytest.mark.skip("Fails sometimes, needs to be fixed")
```
[the remark this answers is not in the document]
Comment (yasakova-anastasia) [reply tests/python/rest_api/test_tasks.py:2282]: https://github.com/opencv/cvat/actions/runs/5458669072/jobs/9934058774
https://github.com/opencv/cvat/actions/runs/5465392239/jobs/9949245380

**Review by kirill-sizov — APPROVED** [review]

**Pull request merged**